### PR TITLE
Fix/secure ingresspattern

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/secure-ingress-auth-cognito/index.ts [hunk 1]
```diff
@@ -120,9 +120,10 @@ class CognitoIdpStack extends cdk.Stack {
         });
 
         // Add the domain to the user pool
+        let r = (Math.random() + 1).toString(36).substring(7);
```
Comment (shapirov103) [inline lib/secure-ingress-auth-cognito/index.ts:123]: i shared from public but it is not correct code. should be 
```
const randomText = 
```
let is only for vars

**Review by shapirov103 — COMMENTED** [review]
minor comment

File: lib/secure-ingress-auth-cognito/index.ts (hunk 1 above)
Comment (elamaran11) [reply lib/secure-ingress-auth-cognito/index.ts]: Fixed

**Comment by elamaran11** [comment]
> minor comment

Fixed it

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**